README.md

```diff
@@ -7,7 +7,7 @@ Use it by setting the extends field in your `.eslintrc`:
 
 ```js
 {
-  extends: '@pmedianetwork/eslint-config',
+  extends: 'pmedianetwork/recommended',
```

Comment:
just a question: why did we get rid of the `@` prefix? Is that somehow required?

Reply:
Plugins for eslint normally use `eslint-plugin-xxx` naming convention. I just sticked with it. Do you think it is required to have it?

Reply:
It would be nice to have all our own dependencies be in the same directory in `node_modules` (makes it easier to look it up, which we sometimes need to do):

![Screenshot 2020-03-05 at 13 07 10](https://user-images.githubusercontent.com/1021430/75980240-5a5ebf00-5ee2-11ea-8845-395bffc7a7df.png)

Reply:
Right - it took me couple of commits so I don't have one to paste, but it's sorted now :)